cfgov/unprocessed/css/organisms/footer.less

```diff
@@ -174,6 +184,14 @@
     }
 
     &-middle-right {
+        .o-footer_list {
+            .respond-to-max( @bp-xs-max, {
```

Comment:
Should the media query go out the outside of this entire block, instead of some class selectors on the outside and some on the inside?

Reply:
Yeah I think that's probably cleaner less, fixed in 534d1f2.